src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/ElementSelector.tsx

```diff
@@ -32,20 +33,24 @@ const ElementSelector = ({
   basePath,
   options,
   value,
+  newValues,
   onChange,
 }: ElementSelectorProps) => {
   return (
     <>
+      Attribute Selector
       <Autocomplete
+        disableClearable={true}
```

Comment:
We could use AutoComplete from Design systems ?

Reply:
Made this change, everything is ruined.

Reply:
What do you mean ? We can't use design-system components ?